### ISSUE TITLE
Inline methods do not recognize signature

### ISSUE THREAD

**Issue by amieres** [fable-compiler/Fable#1238]
### Description

Inline methods do not recognize signature. In the example MethodI (v:bool) is used when it should have been MethodI ().

Also inline produces highly repetitive code with some extra calls to functions.

### Repro code

```
let counter =
    let mutable i = 0
    fun () ->
        i <- i + 1
        i
    
type Type = {
    a : int
    b : int
    c : int
    d : int
    e : int
}
  with
    static member New(n) = {
        a = n
        b = n * 2
        c = n * 3
        d = n * 4
        e = counter()  // <== should only be called twice
      }
      
    member        this.Method  (v:bool) = { this with a = this.a * if v then 2 else 3 }
    member inline this.MethodI (v:bool) = { this with a = this.a * if v then 2 else 3 }
    member        this.Method  ()       = { this with a = this.a * 10 }
    member inline this.MethodI ()       = { this with a = this.a * 10 }

Type.New(5).Method(false).Method(true).Method()
|> printfn "not inline: %A"
    
Type.New(5).MethodI(false).MethodI(true).MethodI()
|> printfn "inline: %A"
```

### Expected and actual results
Expected result (output from FSI):
```
not inline: {a = 300;
 b = 10;
 c = 15;
 d = 20;
 count = 1;}
inline: {a = 300;
 b = 10;
 c = 15;
 d = 20;
 count = 2;}
```

Actual output:
```
not inline: {"a":300,"b":10,"c":15,"d":20,"count":1}
inline: {"a":90,"b":10,"c":15,"d":20,"count":126}
```
notice that when using inline
* a = 90 instead of 300 
* count = 126 instead of 2

### Related information

REPL

**Comment by amieres** [comment]
Nice! 
Thank you
Is the REPL also updated?

**Comment by alfonsogarciacaro** [comment]
Thanks to you @amieres for the detailed report :smile: This should be fixed in dotnet-fable 1.3.1 release. The REPL is being updated on a slower basis, but we're working on a major update that will hopefully also put updates on par with the CLI version.
